test/Microsoft.AspNet.Razor.Test/TagHelpers/TagHelperBlockRewriterTest.cs

```diff
@@ -301,7 +299,20 @@ public class TagHelperBlockRewriterTest : TagHelperRewritingTestBase
                             new MarkupTagHelperBlock("p",
                                 new List<KeyValuePair<string, SyntaxTreeNode>>
                                 {
-                                    new KeyValuePair<string, SyntaxTreeNode>("class", dateTimeNow)
+                                    new KeyValuePair<string, SyntaxTreeNode>(
+                                        "class",
+                                        new MarkupBlock(
+                                            new MarkupBlock(
+                                                new DynamicAttributeBlockChunkGenerator(
+                                                    new LocationTagged<string>(
+                                                        string.Empty,
+                                                        new SourceLocation(9, 0, 9)),
+                                                    new SourceLocation(9, 0, 9)),
```

Comment:
What leads to the new `DynamicAttributeBlockChunkGenerator` with no content? I'm missing the part of this PR that restores child `Chunk`s that were previously removed.